about.html

```diff
@@ -0,0 +1,158 @@
+            <li class="mobile-list"><a href="#services">Our Services</a></li>
+            <hr>
+            <li class="mobile-list"><a href="#contact-me">Our Team</a></li>
+            <hr>
+            <li class="mobile-list"><a href="#supporters">Sponsors</a></li>
+            <hr>
+            <li class="mobile-list"><a href="#">Our Projects</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <section class="about-headline">
+      <article class="about-heading">
+        <h3 class="about-title">Hello Front-End Lovers!</h3>
+        <h1 class="main-heading">Front-End School, best place to learn frontend technologies!</h1>
+      </article>
+
+      <article class="about-intro">
+
+        <p class="about-info">
+          Welcome to our front-end web developer learning pathway!
+          Here we provide you with a structured pathway that will teach you all you need to know to become a front-end
+          web developer. Work through each section, learning new skills (or improving existing ones) as you go along.
+          Each section includes exercises and assessments to test your understanding before you move forward. <br>
+          Choosing the right resources is the key to becoming a successful developer. Most of the beginners follow the
+          wrong resources, and it reults in disappointment. We have chosen best out of best resources available online.
+        </p>
+
+        <p class="reach-us">frontendschool23@gmail.com</p>
+
+      </article>
+    </section>
+
+  </div>
+  <!-- Headline and Navbar End -->
+
+  <!-- About content Start -->
+  <section id="award" class="award">
+    <h3 class="our-award">Front-End School Top Award</h3>
+    <div class="line"></div>
+    <p class="award-txt">Lorem ipsum dolor sit amet consectetur adipisicing elit. Laboriosam veniam ex tempora amet ad
+      placeat optio eligendi consequatur! Odio ex delectus laboriosam. Molestias aspernatur perferendis beatae
+      laboriosam ipsa officiis vero.</p>
+
+    <img class="award-logo" src="./Images/award.jpg" alt="Award image">
+  </section>
+
+  <section id="pastAwards" class="past-awards">
+    <article>
+      <h2>Kindly take a look to our previous awards</h2>
+      <div class="line"></div>
+      <p class="past-info">Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptate, nihil sequi! Adipisci,
+        quis? Dicta autem
+        nisi voluptatum blanditiis commodi nulla. Laudantium, cupiditate mollitia tempora vero tenetur assumenda. Enim,
+        reprehenderit voluptatem?</p>
+    </article>
+
+    <article class="award-boxes">
+      <div class="one">
+        <h3 class="award-date">2018</h3>
+        <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Consequuntur, labore!</p>
+      </div>
+
+      <div class="two">
+        <h3 class="award-date">2021</h3>
+        <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Consequuntur, labore!</p>
+      </div>
+
+    </article>
```

Comment:
I love these cards, but make sure to use the white color for the text which will look much clear.
![image](https://user-images.githubusercontent.com/34779928/205145794-d1e38435-d1ed-474e-8edf-7184d5c042b1.png)